packages/gestalt/src/Toast.js

```diff
@@ -8,7 +8,7 @@ import Icon from './Icon.js';
 
 type Props = {|
   color?: 'darkGray' | 'orange',
-  icon?: 'arrow-circle-forward', // leaving open to additional icons in the future
+  icon?: ?'arrow-circle-forward', // leaving open to additional icons in the future
```

Comment:
As a reference https://wietse.loves.engineering/flowtype-understanding-optionals-and-null-2fa7b98210ce

@chrislloyd @pfarejowicz how do you both feel about this?

Reply:
Personally if neither `icon?: 'arrow-circle-forward'` nor `icon: ?'arrow-circle-forward' work for you, id rather do `icon?: 'arrow-circle-forward' | null`, but im curious what the reason for this change is, do we have a conflict in pinboard that requires `null` to be acceptable

Reply:
For the example of `const iconName = href && !imageUrl ? 'arrow-circle-forward' : null;` i think id personally do `const iconName = href && !imageUrl ? 'arrow-circle-forward' : undefined;`